recipes/qutip/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+    name: qutip
+    version: 3.1.0
+
+source:
+    fn: qutip-3.1.0.tar.gz
+    url: http://qutip.org/downloads/3.1.0/qutip-3.1.0.tar.gz
+
+build:
+    number: 0
+    script: $PYTHON setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - python >=2.7
+    - setuptools
+    - cython >=0.15
+    - numpy x.x
+    
+  run:
+    - python
+    - python >=2.7
+    - cython >=0.15
```

Comment:
Question: Is cython *really* a runtime dependency? That is relatively unusual in my experience.